saskatoon/harvest/api.py

```diff
@@ -220,3 +224,176 @@ def list(self, request, *args, **kwargs):
                                  'title': _("New Person")
                                  }
                          })
+
+
+class StatsView(LoginRequiredMixin, generics.ListAPIView):
+    template_name = "app/stats.html"
+    queryset = Harvest.objects.filter(status="Succeeded")
+    filter_backends = (filters.DjangoFilterBackend,)
+    filterset_class = HarvestFilter
+    filterset_fields = ('status', 'season')
+    permission_classes = [IsCoreOrAdmin]
+
+    def get_queryset(self):
+        self.season = self.kwargs.get('season')
+        if self.season is not None:
+            return self.queryset.filter(start_date__year=self.season)
```

Comment:
again, the `season` field in HarvestFilter should already do this for you when calling `stats/?season=<year>`